cita-cli/src/interactive.rs

```diff
@@ -712,6 +729,7 @@ impl GlobalConfig {
             ("completion_style", completion_style),
             ("edit_style", edit_style),
             ("save_private", save_private.as_str()),
+            ("env_variables", env_variables.as_str()),
```

Comment:
Keys don‘t need to be printed here, we can get them with `get` command

Reply:
Already delete them